mis_builder/models/aep.py

```diff
@@ -421,6 +425,10 @@ def f(mo):
                                      (AccountingNone, AccountingNone))
             if field == 'bal':
```

Comment:
We should put above `v = AccountingNone` for avoiding a later error

Reply:
Not sure about this one, once the other error has been fixed.

Reply:
This is still missing

Reply:
Ah yes, I see it now